package.json

```diff
@@ -39,6 +39,7 @@
         "@navikt/fnrvalidator": "^1.1.3",
         "@navikt/navspa": "^5.0.1",
         "axios": "^0.21.2",
+        "caniuse-lite": "^1.0.30001457",
```

Comment:
Hva er denne dratt inn for?

Reply:
Vi fikk ikke deployet til dev uten, depoy feilet. Derfor tok jeg den inn.